src/main/java/taxi/controller/LoginController.java

```diff
@@ -0,0 +1,38 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.exception.AuthenticationException;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+import taxi.service.AuthenticationServiceImpl;
+
+public class LoginController extends HttpServlet {
+    private final AuthenticationService authenticationService = new AuthenticationServiceImpl();
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/login.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        try {
+            Driver driver = authenticationService.login(login, password);
+            HttpSession session = req.getSession();
+            session.setAttribute("driver_id", driver.getId());
+            resp.sendRedirect("/index");

```

Comment:
fix it everywhere
```suggestion
            resp.sendRedirect(req.getContextPath() + "/index");
```
<img width="942" alt="Знімок екрана 2023-08-13 о 12 23 17" src="https://github.com/mate-academy/jv-web-security/assets/41166578/007950c4-c9a1-486d-b142-b0458cc28cb1">